package.json

```diff
@@ -28,8 +28,9 @@
     "build:release": "polkadot-ci-ghact-build",
     "build:rollup": "polkadot-exec-rollup --config",
     "build:ui": "cd packages/extension && NODE_ENV=production yarn polkadot-exec-webpack --config webpack.extension.cjs --mode production",
-    "build:zip": "yarn build:zip:dst && yarn build:zip:src",
-    "build:zip:dst": "rm -rf ./master-build.zip && cd packages/extension/build && zip -r -FS ../../../master-build.zip .",
+    "build:zip": "yarn build:zip:dst:chrome && yarn adapt-build-to-ff && yarn build:zip:dst:firefox && yarn build:zip:src",
+    "build:zip:dst:chrome": "rm -rf ./master-build-chrome.zip && cd packages/extension/build && zip -r -FS ../../../master-build-chrome.zip .",
+    "build:zip:dst:firefox": "rm -rf ./master-build-firefox.zip && cd packages/extension/build && zip -r -FS ../../../master-build-firefox.zip .",
```

Comment:
does this work in FF? Shouldn't it be zipped like this?
```bash
zip -r -FS ../../../master-build-firefox.zip *"
```
it worked before, but perhaps manifest v3 is more restrictive?